core/templates/dev/head/pages/exploration_editor/translation_tab/translator_overview_directive.html

```diff
@@ -47,4 +47,29 @@
     padding: 2px;
     border-radius: 8px;
   }
+  /* Firefox rules */
+  @-moz-document url-prefix(){
+    .oppia-translation-progress {
+      display: inline-flex;
+      margin-top: 8px;
+    }
+    .oppia-translator-overview{
+      width:650px;
```

Comment:
Style: add space after ":". Also within each rule please alphabetize -- height, then margin, then padding-top, etc.; ditto elsewhere.

Reply:
alphabetise from a to z?